scripts/hfx_filename

```diff
@@ -31,15 +31,11 @@ def rpc(session_id, request):
           "content-length:%d" % (len(request)),
```

Comment:
Request should be converted to binary before getting the length.

Reply:
Thank you for your review.
Actually the unittest tested the function, but didn't consider the non-ascii inupt.
I added a test case for international characters so that they can be covered by the unittest:

> AssertionError: Calls not found.
> Expected: [call(b'POST /remote_db_access?session_id=0 HTTP/1.0\r\n'),
>  call(b'Connection:close\r\n'),
>  call(b'content-length:9\r\n'),
>  call(b'\r\n'),
>  call(b'socket \xe8\xaf\xb7\xe6\xb1\x82')]
> Actual: [call(b'POST /remote_db_access?session_id=0 HTTP/1.0\r\n'),
>  call(b'Connection:close\r\n'),
>  call(b'content-length:13\r\n'),
>  call(b'\r\n'),
>  call(b'socket \xe8\xaf\xb7\xe6\xb1\x82')]